java/frontend/Dockerfile

```diff
@@ -1,18 +1,14 @@
-FROM gradle:7.0.1-jdk11 AS build
-WORKDIR /home/gradle/
-RUN mkdir ./libs
-ENV OTEL_AGENT_VER=1.21.0
-ENV JAVA_AGENT=opentelemetry-javaagent.jar
-ADD https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v${OTEL_AGENT_VER}/${JAVA_AGENT} ./libs

```

Comment:
we don't need to download the agent here because `compileJava` in the `build.gradle` depends on downloading the agent!